assets/training/vision/components/image_classification/spec.yaml

```diff
@@ -188,7 +188,7 @@ outputs:
 
 code: ../src
 
-environment: azureml:AzureML-AutoML-DNN-Vision-GPU:115
```

Comment:
Version 118 is being rolled out now, that has 1.52.0.post1. Could you please use that instead

Reply:
Done